### ISSUE TITLE
Map scroll on interferes with homepage scroll

### ISSUE THREAD

**Issue by darmenise** [publicv-org/publicv#152]
When the user attempts to scroll down on the homepage, they actually scroll down the map. Scrolling the map requires better UI, such as embedded google maps. Try, for instance, https://corsidia.com/schools on both mobile and desktop.

**Another possible solution** would be to adopt the same solution we are developing for Corsidia, which looks like this:

![CleanShot 2021-07-04 at 18 41 43@2x](https://user-images.githubusercontent.com/794154/124391087-8f0da100-dcf7-11eb-9b3c-b0bbcb2acd07.png)

Basically, the map should be vertically shorter, so that the user always has a good area to scroll down. Scrolling down on the map will scroll the map down, while scrolling down on the area below (see the red arrow) will scroll the page down.
